index.html

```diff
@@ -4,27 +4,133 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta name="description" content="An aspiring software developer with experience in desktop, web & mobile applications.">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Audiowide|Sofia|Trirong|Crete Round|Inter">
         <title>Portfolio</title>
         <link rel="stylesheet" href="styles.css">
     </head>
     <body>
-        <header>
+        <div id="home-screen">
             <nav id="navbar">
                 <a href="">Md Tariq-ul Islam</a>
                 <a href=""><img src="./icons/Menu.svg"></a>
             </nav>
-        </header>
-        <div id="welcome">          
-            <h1>Hey There.</br>I'm Tariq-ul Islam</h1>
-            <h2>I am a Software Developer</h2>
-            <p>I can help you build a product, feature or website look through some of my work and experience. If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
-            <ul id="social-media">
-                <li><a href="https://github.com/"><img src="./icons/github.svg"></a></li>
-                <li><a href="https://www.linkedin.com/"><img src="./icons/Linkedin.svg"></a></li>
-                <li><a href="https://angel.co/"><img src="./icons/Angelist.svg"></a></li>
-                <li><a href="https://twitter.com/"><img src="./icons/twitter.svg"></a></li>
-                <li><a href="https://medium.com/"><img src="./icons/medium.svg"></a></li>
-            </ul>
+            <div id="welcome">          
+                <h1>Hey There.</br>I'm Tariq</h1>
+                <h4>I am a Software Developer</h4>
+                <p>I can help you build a product, feature or website look through some of my work and experience. If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+                <ul id="social-media">
+                    <li><a href="https://github.com/"><img src="./icons/github.svg"></a></li>
+                    <li><a href="https://www.linkedin.com/"><img src="./icons/Linkedin.svg"></a></li>
+                    <li><a href="https://angel.co/"><img src="./icons/Angelist.svg"></a></li>
+                    <li><a href="https://twitter.com/"><img src="./icons/twitter.svg"></a></li>
+                    <li><a href="https://medium.com/"><img src="./icons/medium.svg"></a></li>
+                </ul>
+            </div>
         </div>
+        <section id="main-works">
+            <h2 id="heading">My Recent Works</h2>
+            <article class="works">
+                <div class="picture" >
+                    <img src="./images/work1.jpg" alt="My First Project">
```

Comment:
As the previous reviewer mentioned. Kindly Also note that it's a requirement that you use the colors, and images from the Figma design as much as possible. **Your colors and pictures** are currently differing.
Try changing the path for `src="./images/Work1.jpg"` it's should work on that way